TimJohnson/Homework_Week1.js

```diff
@@ -0,0 +1,139 @@
+1a)	"1" == 1; 	true;  using == means equal to and will cast the 1 on the right side of the equation as a string
+1b)	"1" === 1;  false;  using === means that the comparison includes not only equal value but equal type.  (here is string versus number)
+1c)	 x == 'x';	false;  string 'x' is converted tonumber(x) which is different than the number x
+1d)	 x == (x+''); true;  number is equal to that number + an empty string
```

Comment:
Correct, but like above, number is not going to the determining factor. Whatever x contains (except for a few cases, specified in the homework solutions) x is going to be equivalent to itself with an empty string attached.